components/config-mgmt-service/backend/postgres/schema/sql/02_add_rollouts.up.sql

```diff
@@ -18,10 +18,10 @@ CREATE TABLE IF NOT EXISTS rollouts (
   end_time                  TIMESTAMP
 );
 
-CREATE INDEX policy_name_idx        ON rollouts (policy_name);
-CREATE INDEX policy_node_group_idx  ON rollouts (policy_node_group);
-CREATE INDEX policy_revision_id_idx ON rollouts (policy_revision_id);
-CREATE INDEX policy_domain_url_idx  ON rollouts (policy_domain_url);
+CREATE INDEX IF NOT EXISTS policy_name_idx        ON rollouts (policy_name);
+CREATE INDEX IF NOT EXISTS policy_node_group_idx  ON rollouts (policy_node_group);
+CREATE INDEX IF NOT EXISTS policy_revision_id_idx ON rollouts (policy_revision_id);
+CREATE INDEX IF NOT EXISTS policy_domain_url_idx  ON rollouts (policy_domain_url);
```

Comment:
Are some of these changes are from #4183? Is this because the PR has not been rebased?

Reply:
These are in the right place :) If you look at the commits-view, you can see that I 1) added the destructive double migrator thing, 2) changed config-mgmt to use it and fixed the problems it uncovered, 3) added the destructo migrator to applications service (which didn't have any errors)